app/routes/__frontend/events/index.jsx

```diff
@@ -51,6 +51,23 @@ export default function EventsIndex() {
 						</p>
 					</div>
 
+					<h2>Recorded Events</h2>
+					<div className="mb-3 lead">
+						<p>
+							If you can't join our events live, we got you covered! Some of our
+							events, such as Lunch & Learns, are recorded and uploaded to our{' '}
+							<a href="https://www.youtube.com/channel/UCc0579aGEy7jTAgRglR4J0g">
+								YouTube channel
+							</a>
+							.
```

Comment:
- Change the link to `https://www.youtube.com/@VirtualCoffeeIO`
- Let's link to some of our playlists (L&Ls, lightning talks, live streams) in this paragraph.
  
  Maybe something like this:
  
  > If you can't join our events live, we got you covered! You can find [Lunch & Learns](#), our [Live Streams](#) our [2021 Lightning Talks](#), and lots more all on our [YouTube Channel](https://www.youtube.com/@VirtualCoffeeIO).

Reply:
I've changed the link to our YouTube channel and link the playlists 😊